src/libpgmoneta/prometheus.c

```diff
@@ -657,9 +657,14 @@ general_information(int client_fd)
    data = pgmoneta_append(data, "#HELP pgmoneta_retention The retention of pgmoneta\n");
    data = pgmoneta_append(data, "#TYPE pgmoneta_retention gauge\n");
    data = pgmoneta_append(data, "pgmoneta_retention ");
-   data = pgmoneta_append_int(data, config->retention);
+   data = pgmoneta_append_int(data, config->retention_days);
+   data = pgmoneta_append(data, ", ");
+   data = pgmoneta_append_int(data, config->retention_weeks);
+   data = pgmoneta_append(data, ", ");
+   data = pgmoneta_append_int(data, config->retention_months);
+   data = pgmoneta_append(data, ", ");
+   data = pgmoneta_append_int(data, config->retention_years);
```

Comment:
Split into different metrics

Reply:
you mean like "days: 7" instead of plain "7"?

Reply:
change the Prometheus key

Reply:
which basically means default is 0 for the rest ... but lets see

Reply:
I'm not very familiar with prometheus configuration, I'll look into it